config/default/kustomization.yaml

```diff
@@ -1,12 +1,12 @@
 # Adds namespace to all resources.
-namespace: spanner-autoscaler
+namespace: spanner-autoscaler-system
```

Comment:
Is this change required?

Reply:
This was auto generated. I have now reverted it to the original one.